app/src/test/java/com/github/se/polyfit/viewmodel/meal/MealViewModelTest.kt

```diff
@@ -74,6 +74,25 @@ class MealViewModelTest {
     verify { mealFirebaseRepository.getMeal(firebaseID) }
   }
 
+  @Test
+  fun `setMealData sets meal`() {
+    val meal =
+        Meal(
+            MealOccasion.BREAKFAST,
+            "testMeal",
+            1,
+            20.0,
+            NutritionalInformation(mutableListOf()),
+            mutableListOf(),
+            "testID")
+    mealViewModel = MealViewModel(userID, mealRepo = mealFirebaseRepository)
+
+    mealViewModel.setMealData(meal)
+
+    val updatedMeal = mealViewModel.meal.value!!
+    assertEquals(meal, updatedMeal)
+  }
+
```

Comment:
Oops sorry, i changed the meal viewmodel test and the line coverage can be recognized in jacoco now. So you would need to rebase and add this test into the new format